modules/proto/rollkitsdk/sequencer/v1/tx.proto

```
@@ -1,53 +1,28 @@
 syntax = "proto3";
-package rollkitsdk.sequencer;
+package rollkitsdk.sequencer.v1;
 
 import "amino/amino.proto";
```

Buf (1.54.0)

import "amino/amino.proto": file does not exist

(COMPILE)